modules/aws-exfiltration-protection/providers.tf

```diff
@@ -0,0 +1,12 @@
+provider "aws" {
+  region = var.region
+}
+
+// initialize provider in "MWS" mode to provision new workspace
+provider "databricks" {
+  alias      = "mws"
+  host       = "https://accounts.cloud.databricks.com"
+  account_id = var.databricks_account_id
+  username   = var.databricks_account_username
+  password   = var.databricks_account_password
+}
```

Comment:
It's better not to do this in the modules. Providers is recommended to pass explicitly configured.

https://developer.hashicorp.com/terraform/language/meta-arguments/module-providers

Reply:
Moved providers definition from the module to the examples